HeterogeneousCore/CUDACore/interface/CUDAESProduct.h

```diff
@@ -15,11 +15,13 @@
 template <typename T>
 class CUDAESProduct {
 public:
-  CUDAESProduct(): gpuDataPerDevice_(numberOfCUDADevices()) {
-    for(size_t i=0; i<gpuDataPerDevice_.size(); ++i) {
-      gpuDataPerDevice_[i].m_event = std::make_unique<cuda::event_t>(cuda::event::create(cuda::device::current::get_id(),
-                                                                                         cuda::event::sync_by_busy_waiting, // default; we should try to avoid explicit synchronization, so maybe the value doesn't matter much?
-                                                                                         cuda::event::dont_record_timings)); // it should be a bit faster to ignore timings
+  CUDAESProduct() : gpuDataPerDevice_(numberOfCUDADevices()) {
+    for (size_t i = 0; i < gpuDataPerDevice_.size(); ++i) {
+      gpuDataPerDevice_[i].m_event = std::make_unique<cuda::event_t>(cuda::event::create(
+          cuda::device::current::get_id(),
+          cuda::event::
```

Comment:
this is a really ugly change :(

Reply:
Can be likely (mostly) fixed by moving the comments.